### PR TITLE
increase batch memory

### PR DESCRIPTION
**Pull request by srm78**
### Reviewers
**Functional:** 

**Readability:** 

---

## Changes
- add
- remove
- modify

### CODE REVIEW

File: .happy/terraform/modules/batch/main.tf [hunk 1]
```diff
@@ -10,7 +10,7 @@ resource aws_batch_job_definition batch_job_def {
 {
   "jobRoleArn": "${var.batch_role_arn}",
   "image": "${var.image}",
-  "memory": 28000,
+  "memory": 230000,
```
Comment (jgadling) [inline .happy/terraform/modules/batch/main.tf:13]: 1) Can/should we make this configurable per environment?
2) Let's update the vcpu setting to match the old config as well.

Comment (srm78) [reply .happy/terraform/modules/batch/main.tf:13]: vcpu setting is same as old config. i'll make this configurable per environment.

Comment (maniarathi) [reply .happy/terraform/modules/batch/main.tf:13]: Random question -- is this field not deprecated? https://docs.aws.amazon.com/batch/latest/APIReference/API_ContainerProperties.html#Batch-Type-ContainerProperties-memory

Comment (srm78) [reply .happy/terraform/modules/batch/main.tf]: @maniarathi for ec2 resources it is still supported. not for fargate though.

File: .happy/terraform/modules/batch/main.tf [hunk 2]
```diff
@@ -10,7 +10,7 @@ resource aws_batch_job_definition batch_job_def {
 {
   "jobRoleArn": "${var.batch_role_arn}",
   "image": "${var.image}",
-  "memory": 28000,
+  "memory": "${var.batch_container_memory_limit}",
```
Comment (jgadling) [inline .happy/terraform/modules/batch/main.tf:13]: should this be an int or a string? can we remove the quotes?

**Review by jgadling — APPROVED** [review]
Approved once these changes are tested.

File: .happy/terraform/modules/batch/main.tf (hunk 2 above)
Comment (srm78) [reply .happy/terraform/modules/batch/main.tf:13]: yes. quotes needs to be removed. will test on rdev and update the PR.

**Pull request merged**